index.yaml

```diff
@@ -3376,13 +3391,13 @@ entries:
       catalog.cattle.io/release-name: rancher-webhook
     apiVersion: v2
     appVersion: 0.1.0
-    created: "2021-06-11T16:26:33.680221-07:00"
+    created: "2021-06-21T15:12:36.328532-07:00"
     description: ValidatingAdmissionWebhook for Rancher types
-    digest: de5fa7d48cd66c158f30dd02a3ef0c3b1ed6e431e31bf5b9929141a6a0af0e45
+    digest: 3c20cc0a6b0fdc9672d7fa84be46f74b208a00664397d1abcfd8acd9ae10ff0e
     name: rancher-webhook
     urls:
-    - assets/rancher-webhook/rancher-webhook-0.1.0.tgz
-    version: 0.1.0
+    - assets/rancher-webhook/rancher-webhook-0.1.000.tgz
+    version: 0.1.000
```

Comment:
It seems like dev-v2.5 had an incorrect index.yaml for quite a bit of time...

https://github.com/rancher/charts/blob/dev-v2.5/index.yaml#L3371-L3385 shows that dev-v2.5 is missing 0.1.000, which exists in release-v2.5 https://github.com/rancher/charts/blob/037989f43e8090a362f86e3ad6a21c981337a0f8/index.yaml#L3129-L3143. It replaces 0.1.000 with 0.1.0, which was incorrect.

The current PR rewrites 0.1.0 => 0.1.000 and then adds the new version, so the changes in the index.yaml also look correct here to rectify a prior mistake.

cc: @jiaqiluo not sure if there was a missing step here during a migration.